monitoringclient-install.plist

```diff
@@ -30,9 +30,15 @@ GROUP_NAME="ENTER_GROUP_NAME"
 exit 0
 	]]></string>
 	<key>installcheck_script</key>
-	<string><![CDATA[#!/bin/bash
+	<string><![CDATA[#!/bin/zsh
+autoload is-at-least
 wmsubdomain="$(defaults read /Library/MonitoringClient/ClientSettings Subdomain)"
-if [ -e /Library/MonitoringClient/RunClient ] && [[ $wmsubdomain == 'YOURSUBDOMAIN' ]]; then
+wmversion="$(defaults read /Library/MonitoringClient/ClientSettings Client_Version)"
+wmversionmimimum="7.0"
+if [ -e /Library/MonitoringClient/RunClient ] && 
+   [[ $wmsubdomain == "YOURSUBDOMAIN" ]] && 
```

Comment:
If we're going to use `YOURSUBDOMAIN` twice, let's declare it as a variable.

(Like we do with `GROUP_NAME` up above.. but I don't know if changing line 24 to `export GROUP_NAME=…` would carry down to the zsh subshell)